### DISCUSSION TITLE
RustDesk translation

### DISCUSSION THREAD

**Discussion by BestiaPL** [rustdesk/rustdesk#3822] [General]
Hello,
Using version 1.1.9 for Windows x64, I noticed that the Polish language cannot be selected in the GUI. I checked the program's sources and translated the missing texts.

As a result, I have a few questions:
1) in the SRC\LANG\PL.RS I found some untranslated phrases into Polish. I could not find their English (original) meanings anywhere - I had to use translations into other languages (I also checked the TEMPLATE.RS file - apparently the texts were overlooked or were quite old). Can you change the generation of the application to add all languages - and display the untranslated phrases in English and mark in the language files that the text should be quickly completed?
2) in the DOCS folder there are several files in different languages (I am in the process of translation) - what folders still have files that need to be translated?
3) NSIS program installer - I noticed in the SETUP.NSI script that MESSAGEBOX in only in English - will it be possible to translate it into other languages?

**Comment by yurtpage** [comment]
> NSIS program installer

See my PR on translation https://github.com/rustdesk/rustdesk/pull/11585/files
Please note that the file should be not in UTF8 encoding but in Windows-1250 for the Polish.
